js-miniapp-sdk/src/miniapp.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * A module layer for webapps and mobile native interaction.
+ */
+interface MiniAppInterface {
+  /** @returns The Promise of provided id of mini app from injected side. */
+  getUniqueId(): Promise<string>;
+}
+
+/* tslint:disable:no-any */
+export class MiniApp implements MiniAppInterface {
+  getUniqueId(): Promise<string> {
+    return (window as any).MiniAppBridge.getUniqueId();
+  }
+}
+
+module.exports = MiniApp;
```

Comment:
I think this should be `export default MiniApp`.

Reply:
`MiniApp` here is the class, not the instance, so the correct one is `module.exports =`

Reply:
`module.exports` is the es6 style of exporting. Let's use the TypeScript style instead. The compiler will ensure that it's compatible with es5/es6.

Reply:
Oh sorry, I got that wrong. `module.exports` is the commonjs/node way of exporting. Es6 uses the same style as Typescript (`export default`), so we should use that instead.

Reply:
@corycaywood do you mean that we should change commonjs to es6?

Reply:
Yeah, I think we should use the es6/TypeScript style. Because TypeScript will compile it to the correct form.

Reply:
When I tried to build a sample app, it has no problem with commonjs style 🤔 

On my latest commit, I have converted `commonjs` module into `es6`. My sample app is working well with it but please note that it requires `esmify`, `browser-resolve` if using `browserify` in sample app.